nf_core/pipeline-template/nextflow.config

```
@@ -39,7 +39,9 @@ params {
     {%- endif %}
     monochrome_logs              = false
     hook_url                     = null
-    help                         = false
+    {% if nf_schema %}help                         = false
+    helpFull                     = false
+    showHidden                   = false{% endif %}
```

```suggestion
    help_full                    = false
    show_hidden                  = false{% endif %}
```
params are snake_case

---

or is that needed for nf-schema?

---

nf-schema uses `helpFull` and `showHidden` but we can update these defaults via the config if it makes more sense